src/pages/instant-observability.js

```diff
@@ -747,6 +735,19 @@ const QuickstartsPage = ({ data, location }) => {
             )}
             {filteredQuickstarts.map((pack) => (
               <PackTile
+                css={css`
+                  @media screen and (max-width: ${COLLAPSE_BREAKPOINT}) {
+                    flex-direction: row;
+
+                    img {
+                      display: none;
+                    }
+                    .tag-div {
```

Comment:
This seems like something that could be broken out into the `PackTile` component itself, instead of passing it down via css classes. The breakpoints should work there as well